examples/runtime/starlark/README.md

```diff
@@ -0,0 +1,17 @@
+# Starlark Runtime
+
+The `starlark` function provides a runtime to execute a starlark script that
+updates or validate KRM resources.
+
+Starlark source can be provided using `source` field.
```

Comment:
Talked about this bunch of times now: but we need to have a clear delineation between function help and examples.

The README is an example of using Starlark, but is describing a field called `source`, so it seems like a help page. But even then, it's not actually talking what custom resource is being described here (i.e. `StarlarkFunction`), so it's incomplete and ambigious.

Please make sure:
a) Examples README should talk about example. It's fine to explain how this particular example works (e.g. it could use the inline method).
b) Help page should be comprehensive and describe all the details and sematics of each field.

Reply:
+1